I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/add.rpy", line 3, in script
    init python:
  File "game/add.rpy", line 3, in script
    init python:
  File "game/add.rpy", line 42, in <module>
    InventoryItemDict ['alch_fire'] = GetItemDescrText('alch_fire')
NameError: name 'GetItemDescrText' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 326, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 515, in main
    renpy.game.context().run(node)
  File "game/add.rpy", line 3, in script
    init python:
  File "game/add.rpy", line 3, in script
    init python:
  File "renpy/ast.py", line 923, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2235, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/add.rpy", line 42, in <module>
    InventoryItemDict ['alch_fire'] = GetItemDescrText('alch_fire')
NameError: name 'GetItemDescrText' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.8.1895
randal_may_monastry 1.0
Fri Oct 15 01:32:16 2021
